instance/models/openedx_appserver.py

```diff
@@ -241,12 +253,14 @@ def github_admin_username_list(self):
         """
         Returns the github usernames of this instance admins
 
-        Admins are the members of the default team of the `github_admin_organization_name` org
+        Admins are all users listed in github_admin_users and the members of the default team of the
+        organizations in github_admin_organizations.
         """
-        if self.github_admin_organization_name:
-            return get_username_list_from_team(self.github_admin_organization_name)
-        else:
-            return []
+        admin_users = []
```

Comment:
Nit: It shouldn't make a difference in usage, but it's possible for a user to be added multiple times if she exists in more than one organization, or if she's manually specified in addition to belonging to a member organization.